worker/remoterelations/remoterelations.go

```diff
@@ -21,23 +21,31 @@ import (
 
 var logger = loggo.GetLogger("juju.worker.remoterelations")
 
-// RemoteRelationChangePublisherCloser implements RemoteRelationChangePublisher
+// RemoteModelRelationsFacadeCloser implements RemoteModelRelationsFacade
 // and add a Close() method.
-type RemoteRelationChangePublisherCloser interface {
+type RemoteModelRelationsFacadeCloser interface {
 	io.Closer
-	RemoteRelationChangePublisher
+	RemoteModelRelationsFacade
 }
 
-// RemoteRelationChangePublisher instances publish local relation changes to the
-// model hosting the remote application involved in the relation
-type RemoteRelationChangePublisher interface {
+// RemoteModelRelationsFacade instances publish local relation changes to the
+// model hosting the remote application involved in the relation, and also watches
+// for remote relation changes which are then pushed to the local model.
+type RemoteModelRelationsFacade interface {
 	// RegisterRemoteRelation sets up the local model to participate
 	// in the specified relations.
 	RegisterRemoteRelations(relations ...params.RegisterRemoteRelation) ([]params.RemoteEntityIdResult, error)
 
 	// PublishLocalRelationChange publishes local relation changes to the
 	// model hosting the remote application involved in the relation.
 	PublishLocalRelationChange(params.RemoteRelationChangeEvent) error
+
+	// WatchRemoteRelationUnits returns a watcher that notifies of localChanges to the
```

Comment:
I hear you. It is watching for changes in the local model for relations which happen to be remote because they span models.